mteb/tasks/Retrieval/multilingual/BelebeleRetrieval.py

```diff
@@ -2,133 +2,152 @@
-    "tel": ["tel-Telu"],
-    "tgk": ["tgk-Cyrl"],
-    "tgl": ["tgl-Latn"],
-    "tha": ["tha-Thai"],
-    "tir": ["tir-Ethi"],
-    "tsn": ["tsn-Latn"],
-    "tso": ["tso-Latn"],
-    "tur": ["tur-Latn"],
-    "ukr": ["ukr-Cyrl"],
-    "urd": ["urd-Arab"],
-    "uzn": ["uzn-Latn"],
-    "vie": ["vie-Latn"],
-    "war": ["war-Latn"],
-    "wol": ["wol-Latn"],
-    "xho": ["xho-Latn"],
-    "yor": ["yor-Latn"],
-    "zho": ["zho-Hans"],
-    "zsm": ["zsm-Latn"],
-    "zul": ["zul-Latn"],
-}
-
-
-class BelebeleRetrieval(MultilingualTask, AbsTaskRetrieval):
+_LANGUAGES = [
+    "acm_Arab",
+    "afr_Latn",
+    "als_Latn",
+    "amh_Ethi",
+    "apc_Arab",
+    "arb_Arab",
+    "arb_Latn",
+    "ars_Arab",
+    "ary_Arab",
+    "arz_Arab",
+    "asm_Beng",
+    "azj_Latn",
+    "bam_Latn",
+    "ben_Beng",
+    "ben_Latn",
+    "bod_Tibt",
+    "bul_Cyrl",
+    "cat_Latn",
+    "ceb_Latn",
+    "ces_Latn",
+    "ckb_Arab",
+    "dan_Latn",
+    "deu_Latn",
+    "ell_Grek",
+    "eng_Latn",
+    "est_Latn",
+    "eus_Latn",
+    "fin_Latn",
+    "fra_Latn",
+    "fuv_Latn",
+    "gaz_Latn",
+    "grn_Latn",
+    "guj_Gujr",
+    "hat_Latn",
+    "hau_Latn",
+    "heb_Hebr",
+    "hin_Deva",
+    "hin_Latn",
+    "hrv_Latn",
+    "hun_Latn",
+    "hye_Armn",
+    "ibo_Latn",
+    "ilo_Latn",
+    "ind_Latn",
+    "isl_Latn",
+    "ita_Latn",
+    "jav_Latn",
+    "jpn_Jpan",
+    "kac_Latn",
+    "kan_Knda",
+    "kat_Geor",
+    "kaz_Cyrl",
+    "kea_Latn",
+    "khk_Cyrl",
+    "khm_Khmr",
+    "kin_Latn",
+    "kir_Cyrl",
+    "kor_Hang",
+    "lao_Laoo",
+    "lin_Latn",
+    "lit_Latn",
+    "lug_Latn",
+    "luo_Latn",
+    "lvs_Latn",
+    "mal_Mlym",
+    "mar_Deva",
+    "mkd_Cyrl",
+    "mlt_Latn",
+    "mri_Latn",
+    "mya_Mymr",
+    "nld_Latn",
+    "nob_Latn",
+    "npi_Deva",
+    "npi_Latn",
+    "nso_Latn",
+    "nya_Latn",
+    "ory_Orya",
+    "pan_Guru",
+    "pbt_Arab",
+    "pes_Arab",
+    "plt_Latn",
+    "pol_Latn",
+    "por_Latn",
+    "ron_Latn",
+    "rus_Cyrl",
+    "shn_Mymr",
+    "sin_Latn",
+    "sin_Sinh",
+    "slk_Latn",
+    "slv_Latn",
+    "sna_Latn",
+    "snd_Arab",
+    "som_Latn",
+    "sot_Latn",
+    "spa_Latn",
+    "srp_Cyrl",
+    "ssw_Latn",
+    "sun_Latn",
+    "swe_Latn",
+    "swh_Latn",
+    "tam_Taml",
+    "tel_Telu",
+    "tgk_Cyrl",
+    "tgl_Latn",
+    "tha_Thai",
+    "tir_Ethi",
+    "tsn_Latn",
+    "tso_Latn",
+    "tur_Latn",
+    "ukr_Cyrl",
+    "urd_Arab",
+    "urd_Latn",
+    "uzn_Latn",
+    "vie_Latn",
+    "war_Latn",
+    "wol_Latn",
+    "xho_Latn",
+    "yor_Latn",
+    "zho_Hans",
+    "zho_Hant",
+    "zsm_Latn",
+    "zul_Latn",
+]
+
+
+def get_lang_pairs() -> dict[str, list[str]]:
+    # add all possible language pairs
+    lang_pairs = {}
+    for x in _LANGUAGES:
+        for y in _LANGUAGES:
+            xx = x.replace("_", "-")
+            yy = y.replace("_", "-")
+            pair = f"{xx}-{yy}"
+            lang_pairs[pair] = [xx, yy]
+    return lang_pairs
```

Comment:
All possible pairs might be unreasonable as some pairs are in practice never seen. I believe we have another dataset (Bible I believe), which created a meaningful subset.